modules/user/lib/user-sync-vebal-lock-balance.service.ts

```diff
@@ -0,0 +1,113 @@
+import { UserStakedBalanceService, UserSyncUserBalanceInput } from '../user-types';
+import { prisma } from '../../../prisma/prisma-client';
+import { getContractAt } from '../../web3/contract';
+import _ from 'lodash';
+import { prismaBulkExecuteOperations } from '../../../prisma/prisma-util';
+import RewardsOnlyGaugeAbi from './abi/RewardsOnlyGauge.json';
+import { Multicaller } from '../../web3/multicaller';
+import { formatFixed } from '@ethersproject/bignumber';
+import { PrismaPoolStakingType } from '@prisma/client';
+import { networkContext } from '../../network/network-context.service';
+import { GaugeSubgraphService } from '../../subgraphs/gauge-subgraph/gauge-subgraph.service';
+import { veBalLocksSubgraphService } from '../../subgraphs/veBal-locks-subgraph/veBal-locks-subgraph.service';
+import { BigNumber } from 'ethers';
+import VeBalABI from '../../vebal/abi/vebal.json';
+import mainnet from '../../../config/mainnet';
+
+export class UserSyncVebalLockBalanceService implements UserStakedBalanceService {
+    get chain() {
+        return networkContext.chain;
+    }
```

Comment:
Isn’t that unnecessary since it will only be called in the mainnet configuration?